connectors/aws/aws-textract/src/main/java/io/camunda/connector/textract/TextractConnectorFunction.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ *       under one or more contributor license agreements. Licensed under a proprietary license.
+ *       See the License.txt file for more information. You may not use this file
+ *       except in compliance with the proprietary license.
+ */
```

Comment:
It seems there's something wrong with the license indentation. Could you please check it?

Reply:
I have checked 5 more License.txt files and everywhere same data